### ISSUE TITLE
soroban-cli: error bumping contract (error: jsonrpc error: Parse error: missing field `transactionData` at line 1 column 444)

### ISSUE THREAD

**Issue by leighmcculloch** [stellar/soroban-tools#921]
### What version are you using?

- stellar/soroban-tools@b1519a4d90502ce3155960541b2a98c88f3dd043 (rpc and cli)
- stellar/stellar-core@22b9bb38416e5ce058eb9c1fadfdd414a4661833
- stellar/rs-soroban-env@eb5a9ba053a7b64a8eff605db625525378f7bea0

### What did you do?

```console
❯ soroban lab token wrap --network local --source me --asset USD:GDP7GGFMNMWABGWTF6BUB45MN6TPOCFY43KJVK5LDBCSJ2QTMQYYU3DA
CAW7SH5JG45U6YLCFB6YD2KVSOZQHMNEJQPN6KDZOCGSXWR72ZIFBBMP

❯ soroban contract bump --network local --source me --id CAW7SH5JG45U6YLCFB6YD2KVSOZQHMNEJQPN6KDZOCGSXWR72ZIFBBMP --durability persistent --ledgers-to-expire 1
```

The `local` network is a local quickstart running the commits above.

The `me` source is the networks root account.

### What did you expect to see?

Some sort of success message. Or some sort of meaningful error message.

### What did you see instead?

```console
error: jsonrpc error: Parse error: missing field `transactionData` at line 1 column 444
```

### Discussion

Similar to #917 it seems like the CLI<>RPC API is out of sync.

**Comment by willemneal** [comment]
Fixed by #855

**Comment by xsoul** [comment]
May I know how to fix this issue? currently using soroban 0.9.4 (76cacc7cf885bd7d37756735fab332351d893406)

**Comment by willemneal** [comment]
The current version is 20.0.0-rc4 which should solve this.
